testdata/v1_full/terraform/accounts/bar/terraform.d

```diff
@@ -1 +1 @@
-/Users/rking/go/src/github.com/chanzuckerberg/fogg/testdata/v1_full/terraform.d
\ No newline at end of file
+../../../terraform.d
```

Comment:
this was a bug, fixed now. unfortunately our tests don't catch this because we don't do any assertions on links